nox/sessions.py

```diff
@@ -56,6 +56,16 @@ class _SessionQuit(Exception):
     pass
 
 
+class _SessionSkip(Exception):
+    pass
+
+
+class SessionStatus(object):
+    FAIL = False
+    SUCCESS = True
+    SKIP = 3
```

Comment:
Crap, I meant to make `SKIP = 2`. It's supposed to be `0, 1, 2`, I used True and False for consistency with `success` and also because `False is 0 is True`.